pages/management/SettingsHandler.inc.php

```diff
@@ -37,19 +37,26 @@ function __construct() {
 	}
 
 	/**
-	 * Add the archive and payments tabs to the distribution settings page
+	 * Add the archive and access tabs to the distribution settings page
 	 *
-	 * @param $args array
-	 * @param $request Request
+	 * @copydoc ManagementHandler::_setupDistributionSettingsData()
 	 */
-	function distribution($args, $request) {
-		parent::distribution($args, $request);
+	function _setupDistributionSettingsData($request) {
+  		$settingsData = parent::_setupDistributionsettingsData($request);
 
 		$templateMgr = TemplateManager::getManager($request);
 		$context = $request->getContext();
 		$router = $request->getRouter();
 		$dispatcher = $request->getDispatcher();
 
+		// Hook into the settings templates to add the appropriate tabs
+		HookRegistry::register('Template::Settings::distribution', function($hookName, $args) {
+			$templateMgr = $args[1];
+			$output = &$args[2];
```

Comment:
Preferred syntax is `$a =& $b` rather than `$a = &$b`

Reply:
check